sass/index.scss

```diff
@@ -576,6 +610,23 @@ footer {
             }
         }
     }
+
+    // Contact
+    #contact {
+        flex-direction: column;
+
+        #contact-content {
+            width: 100%;
+            padding-right: 0px;
+        }
+
+        #contact-socials {
+            margin: 2rem auto -2px;
+            max-width: 75%;
+            width: 75%;
+            height: calc((100vw - 10rem) * 0.56); // should be the same as 75% width
```

Comment:
Why the calc vs. 75vw? (And hwere does 0.56 come from?)

Reply:
In the image's aspect ratio and original desktop styling, the height is ~75% of the width. Lines 367 - 369:

`width: 430px;`
`max-width: 50%;`
`height: 330px;`

I want to maintain this aspect ratio when the width is set to 75% as it is here. There's a few steps that it takes to get to the right number

75% width is not actually 75vw here. It's 75% of the space that the container is allowed to fill. So 100vw minus the padding for the section, which is a total of 7rem. (I mistakenly put 10 in the code here because the desktop section padding adds up to 10rem in width. I will be changing that after I post this comment.) So, to get the 100% value we do 100vw - 7rem

Then, we need to multiply by 75% to get the width of the image. Then by 75% again to get the height to be 75% of the width value. 0.75 * 0.75 is ~0.56

So in the end, getting roughly the same aspect ratio leaves us with a height of (100vw - 7rem) * 0.56

If you double check my work, keep in mind that the box model breakdown in inspect tool shows width of container and padding as seperate. We have the css set to include the padding as a part of the total width value. As I remembered while trying to go back and check my work to explain it.

Actually changing this to 75vw severely messes with it. 

When I go to fix the 10rem to 7rem, I will try to clarify my comment a bit more.

Reply:
> It's 75% of the space that the container is allowed to fill.
Ahhhhh I see, I forgot how percents worked lol

In that case I'd suggest making a variable or two. This can be done in general PR as you clarified all the logic here so don't consider this to be a blocker: but maybe
```css
height: calc(((100vw - var(--section-pad)) * (0.75 * 0.75))) // squared to match height and width
```
(I don't fully understand the `0.75 * 0.75` but I shall take your word for it)

Or you can just link to this comment; I do that a lot when there's a lot of nuances depending on the current state of the code

Reply:
> I don't fully understand the 0.75 * 0.75

Width of the image is 0.75 * the total possible width (width: 75%)
Height of the image needs to be 75% of the width, aka 0.75 * width aka 0.75 * 0.75 * the total possible width
If that makes sense

The comment in your sample is inaccurate since it's not "to match"

I could add a variable for section padding, but since section padding is probably going to be changed in general pr anyways I agree to save it for then